hotline/notifications/views.py

```diff
@@ -25,3 +25,18 @@ def create(request):
     return render(request, "notifications/create.html", {
         "form": form,
     })
+
+
+@permissions.is_active
+def list_(request):
+    if request.method == "POST":
+        form = UserSubscriptionDeleteForm(request.POST, user=request.user)
+        if form.is_valid():
+            form.save()
```

Comment:
are you suggesting something like

    @permissions.is_active
    def list_(request):
        if request.method == "POST":
            messages.success(request, "saved")
            form = UserSubscriptionDeleteForm(request.POST, user=request.user)
            if form.is_valid():
                form.save()
            ...

instead of

    @permissions.is_active
    def list_(request):
        if request.method == "POST":
            form = UserSubscriptionDeleteForm(request.POST, user=request.user)
            if form.is_valid():
                messages.success(request, "Saved")
                form.save()
            ...

?

Should I add another messages call in the event that the form is invalid then?